app/views/complaints/_complaints_table.html.erb

```diff
@@ -29,6 +29,6 @@
   <!-- this content will change when sort changes -->
 </div>
 
-<%= render partial: 'complaints/pagination', locals: {pagy: @pagy} %>
+<%= render partial: 'complaints/pagination', locals: {pagy: @pagy, path_args: {sort: @sort}} %>
```

Comment:
[thought] I wonder if eventually, assuming we had filters, etc, if we would want to use a clone of `params` rather than listing individual parameters. idk. For now I think this is fine, just thinking out loud.

Reply:
Might be worthwhile later, but I think for now being explicit is better.